plugin/src/main/java/io/fabric8/maven/plugin/converter/DeploymentOpenShiftConverter.java

```diff
@@ -99,7 +101,7 @@ public HasMetadata convert(HasMetadata item) {
                 specBuilder.addNewTrigger().withType("ConfigChange").endTrigger();
 
                 // add a new image change trigger for the build stream
-                if (containerToImageMap.size() != 0) {
+                if (containerToImageMap.size() != 0 && useImageStreamTriggers) {
```

Comment:
my guess would be - to only use them when we've created the image via an ImageStream? I wasn't sure how to reliably determine a container image was created via an ImageStream though or I'd have added that to the PR